tests/accounts/base.py

```diff
@@ -0,0 +1,95 @@
+import base64
+import os
+import uuid
+
+from demands import HTTPServiceError
+from mock import Mock
+from unittest2 import TestCase
+from yoconfig import configure
+from yoconfigurator.base import read_config
+
+
+class BaseTestCase(TestCase):
```

Comment:
@sgalaburda commented on a diff that is now outdated saying that I consider classes with `Base` in the name to be a code smell. @ekoval asked why and I never replied.

Now I am replying. Keep in mind I haven't looked at this code, I'm just explaining my opinion:

It is too generic. Giving something the name "base" makes it easy to become a dumping ground for any functionality you might want somewhere else. So I think it's usually a sign that a more expressive name is needed, or that it needs to be split into smaller, more cohesive pieces that are easily named for what they are.